src/randomThings.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+const botActivities = require("./botActivities");
+let user;
+
+const init = (client) => {
+	user = client.user;
+};
+const randomString = function(array, username = "", setActivity = true) {
+	if (setActivity) setRandomActivity();
+
+	const msg = array[Math.floor(Math.random() * array.length)];
+	if (msg.includes("%s")) return util.format(msg, username);
```

Comment:
util not defined in this file causing program to crash.

Reply:
good catch!